client/state/analytics/actions/enhance-with-site-main-product.js

```diff
@@ -0,0 +1,51 @@
+import { set } from 'lodash';
```

Comment:
Any chance we could avoid introducing new Lodash usage?

We've been trying to get rid of it in the long term: p4TIVU-9Bf-p2